deployability/modules/testing/testing.py

```diff
@@ -6,7 +6,8 @@
 from modules.generic.utils import Utils
 from .models import InputPayload, ExtraVars
 from .utils import logger
-
+import os
+import json
 class Tester:
```

Comment:
Order the imports and add two new lines before the `class` definition (only styling)

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5112/commits/dd818b765d551ab2c6431656986ec60cea05225c